azkaban-web-server/src/main/java/azkaban/imagemgmt/servlets/HPFlowServlet.java

```diff
@@ -0,0 +1,129 @@
+package azkaban.imagemgmt.servlets;
+
+import azkaban.imagemgmt.daos.HPFlowDao;
+import azkaban.imagemgmt.dto.HPFlowDTO;
+import azkaban.imagemgmt.exception.ErrorCode;
+import azkaban.imagemgmt.exception.ImageMgmtException;
+import azkaban.imagemgmt.exception.ImageMgmtInvalidInputException;
+import azkaban.imagemgmt.exception.ImageMgmtInvalidPermissionException;
+import azkaban.imagemgmt.exception.ImageMgmtValidationException;
+import azkaban.imagemgmt.permission.PermissionManager;
+import azkaban.imagemgmt.services.HPFlowService;
+import azkaban.imagemgmt.utils.ConverterUtils;
+import azkaban.server.HttpRequestUtils;
+import azkaban.server.session.Session;
+import azkaban.user.User;
+import azkaban.webapp.AzkabanWebServer;
+import azkaban.webapp.servlet.LoginAbstractAzkabanServlet;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.http.HttpStatus;
+import org.apache.log4j.Logger;
+
+
+/**
+ * This class handles all the REST API calls for API "/hpFlows"
+ * It currently implements addition of high priority flows.
+ */
+public class HPFlowServlet extends LoginAbstractAzkabanServlet {
+
+  private static final Logger log = Logger.getLogger(HPFlowServlet.class);
+  private static final String BASE_HP_FLOW_URI = "/hpFlows";
+  private ConverterUtils converterUtils;
+  private HPFlowService hpFlowService;
+
+  public HPFlowServlet() {
+    super(new ArrayList<>());
+  }
+
+  @Override
+  public void init(final ServletConfig config) throws ServletException {
+    super.init(config);
+    final AzkabanWebServer server = getApplication();
+    this.converterUtils = server.getConverterUtils();
+    this.hpFlowService = server.getHPFlowService();
+  }
+
+  @Override
+  protected void handleGet(final HttpServletRequest req,
```

Comment:
OOC: Is this a todo or do we have no plans to implement the Get call?

Reply:
At the time when this PR was created, expectation was to have basic functionality implemented hence this one was not implemented. However, things have now changed, it will be implemented in the upcoming commit for this PR.